examples/adfox.amp.html

```diff
@@ -0,0 +1,71 @@
+<!doctype html>
```

Comment:
We don't need example page for every ad network. Please only add the example to ads.amp.html page.